src/games/hex.h

```diff
@@ -17,7 +17,8 @@
 
 namespace Hex {
 
-enum Color { COLOR_BLACK, COLOR_WHITE, COLOR_NONE };
+// Niki added the explicit 0
```

Comment:
I don't think you need to add comments stating what you changed, but seems fine.